src/Compiler/Microsoft.AspNetCore.Razor.Language/test/TestFiles/IntegrationTests/ComponentDesignTimeCodeGenerationTest/CascadingGenericInference_ExplicitOverride/TestComponent.codegen.cs

```diff
@@ -60,6 +60,15 @@ protected override void BuildRenderTree(global::Microsoft.AspNetCore.Components.
 #nullable disable
             }
             ));
+            __o = nameof(global::Test.Grid<string>.
```

Comment:
We'll want versions of this test where `string` isn't a valid type argument for `Grid`

Reply:
New test `ChildComponent_MultipleGenerics_TypeContraints` added for this case.